src/apps/replication/client_lib/replication_common.h

```diff
@@ -97,6 +97,7 @@ class replication_options
     int32_t log_file_size_mb;
     int32_t log_batch_buffer_KB_shared;
     int32_t log_batch_buffer_KB_private;
+    // TODO(qinzuoyan): the following log options is not used anywhere
```

Comment:
Not only that, the log related code is some complicated, and some bugs exists, I will look into it, add some comments, fix the bugs.